WordPress/src/jetpack/res/xml/shortcuts.xml

```diff
@@ -1,7 +1,5 @@
 <!--WARNING - build flavors might use a copy of this file -->
-<shortcuts xmlns:android="http://schemas.android.com/apk/res/android"
-           xmlns:tools="http://schemas.android.com/tools"
-           tools:targetApi="n_mr1">
```

Comment:
Warning (⚠️): This change you did he also applies to other such variants, please search for `tools:targetApi="n_mr1"` and apply the same change there too:

- `wasabi/.../shortcuts.xml`
- `jetpackWasabi/.../shortcuts.xml`
- `jalapeno/.../shortcuts.xml`
- `jetpackJalapeno/.../shortcuts.xml`

Reply:
Thanks! Addressed in e42dd6d

Reply:
Awesome, thanks @wzieba ! 👍